web/src/people.ts

```diff
@@ -382,7 +382,9 @@ export function email_list_to_user_ids_string(emails: string[]): string | undefi
         return undefined;
     }
 
-    user_ids = sort_numerically(user_ids);
+    if (sort) {
+        user_ids = sort_numerically(user_ids);
+    }
```

Comment:
This change is not being used anywhere in the PR. Maybe we should remove it?

Reply:
Removed it! Was trying to implement using this function which sorted the results. Ended up not using it for simplicity.